<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>solutioPage</class>
 <widget class="QWidget" name="solutioPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="nextPage">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>710</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 183);
border-color: rgb(170, 170, 127);
font: 16pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>recommencer l'aventure</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>331</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../Downloads/Download Cartoon young farmer with hen and dog for free.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
